client/src/locales/en-US.json

```diff
@@ -392,6 +391,30 @@
             "actionRevoke": "Revoke this user | Revoke these users",
             "actionDetails": "View detail",
             "actionSeeCommonWorkspaces": "See common workspaces"
+        },
+        "greet": {
+            "titles": {
+                "waitForGuest": "Onboard a new user",
+                "provideHostCode": "Share your code",
+                "getGuestCode": "Get guest code",
+                "contactDetails": "Contact details",
+                "summary": "User has been added successfully!"
+            },
+            "subtitles": {
+                "waitForGuest1": "To onboard a new user, you have to be in direct contact with them for the whole duration of the onboarding process.",
```

Comment:
"with him" that's a single user

Reply:
Nop, `him` is masculine, `them` is used as a neutral pronoun.